ADVANCED.md

```diff
@@ -306,54 +305,136 @@ import { SearchBox } from "@elastic/react-search-ui";
 
 ### Example using Autocompleted Suggestions
 
+"Suggestions" are different than "Results". Suggestions are suggested queries. The
+default behavior for suggestions when selected is to perform the search in
+the main search page, refreshing the search results.
+
 ```jsx
+<SearchBox autocompleteSuggestions={true} />
+```
 
-import { SearchBox } from "@elastic/react-search-ui";
+### Example using Autocompleted Suggestions and Autocompleted Results
 
-...
+The default view will show both results and suggestions, divided into
+sections. Section titles can be added to help distinguish between the two.
 
+```jsx
 <SearchBox
-  autocompleteSuggestions={true}
+  autocompleteResults={{
+    sectionTitle: "Suggested Results",
+    titleField: "title",
+    urlField: "nps_link"
+  }}
+  autocompleteSuggestions={{
+    sectionTitle: "Suggested Queries"
+  }}
 />
 ```
 
-### Example using multiple types of Autocompleted Suggestions
+### Configuring autocomplete queries
+
+Autocomplete queries can be customized in the `SearchProvider` configuration, using the `autocompleteQuery` property.
+See the [Advanced Configuration Guide](#advanced-configuration) for more information.
 
 ```jsx
+<SearchProvider
+    config={{
+      ...
+      autocompleteQuery: {
+        // Customize the query for autocompleteResults
+        results: {
+          result_fields: {
+            // Add snippet highlighting
+            title: { snippet: { size: 100, fallback: true } },
+            nps_link: { raw: {} }
+          }
+        },
+        // Customize the query for autocompleteSuggestions
+        suggestions: {
+          types: {
+            // Limit query to only suggest based on "title" field
+            documents: { fields: ["title"] }
+          },
+          // Limit the number of suggestions returned from the server
+          size: 4
+        }
+      }
+    }}
+>
+    <SearchBox
+      autocompleteResults={{
+        sectionTitle: "Suggested Results",
+        titleField: "title",
+        urlField: "nps_link"
+      }}
+      autocompleteSuggestions={{
+        sectionTitle: "Suggested Queries",
```

Comment:
Nitpick: The functions are `autocompleteSuggestions` and `autocompleteResults`. The header says 'autocompleted ...'. Maybe make them consistent?🤔

Reply:
Done